### ISSUE TITLE
Train from scratch (3DGS + model)

### ISSUE THREAD

**Issue by Mariusmarten** [YihangChen-ee/FCGS#15]
Hi, thank you for publishing your code. I have some questions:

1. I want to reproduce the training results on DL3DV. Do you have the training script for the 3DGS? Which scenes were selected as the training (not excluded) and test set?
2. What training script are you running for training across the DL3DV dataset?

**Comment by YihangChen-ee** [comment]
Hi,
1. Please find the attachment for the training set. The testing set can be found [in this issue](https://github.com/YihangChen-ee/FCGS/issues/12).

[training_set.txt](https://github.com/user-attachments/files/20301052/training_set.txt)

2. Training script is not released at this moment. Tips on the training strategy can be found [in this issue](https://github.com/YihangChen-ee/FCGS/issues/14).
